### ISSUE TITLE
Accessibility Issue - Contrast on disabled button

### ISSUE THREAD

**Issue by brandydanner-gsa** [GSA/sam-design-system#620]
**Components Involved**
button: `disabled` with styles classes `usa-button` and `usa-button--secondary`

`<button _ngcontent-cxg-c207="" id="feedbackSubmit" aria-label="Feedback submit" class="usa-button usa-button--secondary" disabled="" data-la-highlighted-by="c30bfba7-f6b6-2308-5693-8a3debc5a077-toggle">Submit</button>`

`<button disabled="" class="usa-button usa-button--secondary">Download</button>`

**Expected Behavior**
Contrast on disabled button meets accessibility requirements

**Actual Behavior**
Contrast on disabled button does not meet accessibility requirements: "This BUTTON contains text with a background color of #c9c9c9 rgb(201, 201, 201) and foreground color of #ffffff rgb(255, 255, 255) that is less than 18 point in size; or bold text less than 14 point in size that has a luminosity contrast ratio of 1.656, which is below 4.5:1"

**Your Environment**
Angular Version: 10.2
@gsa-sam packages version: 10.0.12, sam-styles 0.0.102

**Additional context**

![image](https://user-images.githubusercontent.com/75684861/110367150-4ed23680-8015-11eb-8638-86a1f242f037.png)

Disabled footer feedback submit button:
![image](https://user-images.githubusercontent.com/75684861/110352505-01999900-8004-11eb-9647-1a4aa6038c44.png)

Disabled download button:
![image](https://user-images.githubusercontent.com/75684861/110366598-84c2eb00-8014-11eb-8d7c-79922ff09677.png)

**Comment by Bibek-Regmi** [comment]
@brandydanner-gsa  - This is a false flag by access assistant. The colors used for disabled button here are the same colors used by USWDS - https://designsystem.digital.gov/components/button/
While the access assistant is correct in that the contrast ratio is only 1.656, disabled buttons are seen as an exception to the contrast requirement rules -https://github.com/uswds/uswds/issues/944#issuecomment-169888646

**Comment by brandydanner-gsa** [comment]
Thanks Bibek.  We'll take a note of this and I'll close this Issue.
